signature-aggregator/api/api.go

```diff
@@ -94,25 +106,49 @@ func signatureAggregationAPIHandler(
 		}
 		var decodedMessage []byte
 		decodedMessage, err = hex.DecodeString(
-			strings.TrimPrefix(req.UnsignedMessage, "0x"),
+			strings.TrimPrefix(req.Message, "0x"),
 		)
 		if err != nil {
 			msg := "Could not decode message"
 			logger.Warn(
 				msg,
-				zap.String("msg", req.UnsignedMessage),
+				zap.String("msg", req.Message),
 				zap.Error(err),
 			)
 			writeJSONError(logger, w, msg)
 			return
 		}
-		unsignedMessage, err := types.UnpackWarpMessage(decodedMessage)
+		message, err := types.UnpackWarpMessage(decodedMessage)
 		if err != nil {
 			msg := "Error unpacking warp message"
 			logger.Warn(msg, zap.Error(err))
 			writeJSONError(logger, w, msg)
 			return
 		}
+
+		justification, err := hex.DecodeString(
+			strings.TrimPrefix(req.Justification, "0x"),
```

Comment:
we have `HexOrCB58ToID` and `SanitizieHexString` utils functions that might apply here

Reply:
This doesn't decode to an `ID` but yeah we should call `SanitizeHexString` here

Reply:
3d7707dc370ad953a957f6297b9f03cfd27c9431